corehq/apps/domain/calculations.py

```diff
@@ -67,14 +67,11 @@ def active_mobile_users(domain, *args):
     now = datetime.utcnow()
     then = (now - timedelta(days=30))
 
-    user_ids = get_mobile_users(domain)
-
     form_users = set(
         FormES()
         .domain(domain)
         .user_aggregation()
         .submitted(gte=then)
-        .user_id(user_ids)
```

Comment:
Likewise @emord do you remember what your intention might have been when you added this line during a small rewrite 3 and a half years ago?

Reply:
[That PR](https://github.com/dimagi/commcare-hq/pull/5345) was from my first 6 months at Dimagi and it was the dreaded admin reports.

With that said, I think that this might be a wrong change since this would also include system forms and web user forms (maybe common if the domain uses app preview?). To make it more confusing later this uses `len(get_mobile_users(domain)) - num_users` but I think that would sometimes count those web users in `num_users` meaning there are fewer inactive users reported than we would expect.